src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import AutocompleteOption from '../AutocompleteOption/AutocompleteOption';
+import classes from './Autocomplete.module.scss';
+
+const Autocomplete = ({ display, options, onOptionSelect }) => {
+  // useEffect(() => {}, [display, options]);
```

Comment:
please remove comments

Reply:
done

Reply:
please remove commented code everywhere you have it (if it won't be used anymore)